tools/ashlar/ashlar.xml

```diff
@@ -0,0 +1,184 @@
+<tool id="ashlar" name="ASHLAR" version="@VERSION@.1" profile="17.09">
```

Comment:
I guess the profile version can be increased?
We recommend using TOOL_VERSION https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions 

If you stick to it we have a bot that can bump the requirement versions etc ...